content/kb/tutorials/databases/snowflake.md

```diff
@@ -107,39 +104,39 @@ Copy the code below to your Streamlit app and run it. Make sure to adapt the que
 import streamlit as st
 
 # Initialize connection.
-conn = st.experimental_connection('snowpark')
+conn = st.connection("snowflake", type="snowflake")
```

Comment:
Either remove `type="snowflake"` here (my preference) or else name the connection something else.

Reply:
That was my bad. I missed the line about the type being inferred by the name. Will do.

Reply:
Done in [6f5c73b](https://github.com/streamlit/docs/pull/860/commits/6f5c73b9affdee566e3a4d81f05cac6992deb3e5)